pdns/tcpreceiver.cc

```diff
@@ -23,6 +23,7 @@
 #include "config.h"
 #endif
 #include <boost/algorithm/string.hpp>
+#include <boost/scoped_array.hpp>
```

Comment:
This seems weird to me, why does tcpreceiver suddenly need this include?

Reply:
`tcpreceiver.cc` uses scoped arrays, previously it got the include (indirectly) via `responsestats.hh`, but that one is no longer including it, since it switched to using `std::array`s.

Reply:
thanks!